### PR TITLE
MGMT-10204: Add LVMO to the hub

### PR DESCRIPTION
**Pull request by flaper87**
# Description

Add LVMO to the hub so that we can stop requesting PV creation in advance

## Type of change

Please select the appropriate options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change is a documentation update

## Testing

**Test Configuration**:

- Versions:
- Hardware:

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] If a change is adding a feature, it should require a change to the README.md and the review should catch this.
- [ ] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [ ] A change should not be merged unless it passes CI or there is a comment/update saying what testing was passed.
- [ ] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: deploy-lvmo/deploy-hub.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>"
+        oc --kubeconfig=${KUBECONFIG_HUB} apply -f manifests/04-LVMO-LVMOCluster.yaml
+        sleep 60
+
+        echo ">>>> Waiting for: LVMCluster"
+        echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+        timeout=0
+        ready=false
+        while [ "$timeout" -lt "1000" ]; do
+            if [[ $(oc get --kubeconfig=${KUBECONFIG_HUB} -n openshift-storage lvmcluster -ojsonpath='{.items[*].status.ready}') == "true" ]]; then
```
Comment (eifrach) [inline deploy-lvmo/deploy-hub.sh:44]: This is relating to #538 
maybe we should remove the exit on error

**Review by danielchg — APPROVED** [review]

File: deploy-lvmo/deploy-hub.sh (hunk 1 above)
Comment (flaper87) [reply deploy-lvmo/deploy-hub.sh:44]: I will do this refactor in both places in a follow-up PR

**Comment by flaper87** [comment]
This needs to be documented! I will open a PR for docs once we decide the right approach

**Pull request merged**